cmd/kawad/config.go

```diff
@@ -34,6 +35,9 @@ func init() {
 	loader.Register("printer", func() loader.Builder[kawa.Destination[types.Event]] {
 		return &PrinterConfig{}
 	})
+	loader.Register("sqs", func() loader.Builder[kawa.Destination[[]byte]] {
```

Comment:
This should be `types.Event`, which requires implementing a small shim between bytes and `types.Event`.

See @harpesichord's PR for MQTT for an example:

https://github.com/runreveal/kawa/pull/22/files#diff-c98d0db234d358f55fa681b43ed3ae5cceb2257c3c44c4e77c1475b36af04cc2